pkg/gke/update.go

```diff
@@ -477,7 +478,7 @@ func UpdateNodePoolAutoscaling(
 		needsUpdate = true
 	}
 	if needsUpdate {
-		logrus.Infof("updating autoscaling config of node pool [%s] on cluster [%s]", *nodePool.Name, config.Name)
+		logrus.Infof("updating autoscaling config (upstream: %+v; config: %+v) of node pool [%s] on cluster [%s (%s)]", upstreamNodePool.Autoscaling, nodePool.Autoscaling, *nodePool.Name, config.Spec.ClusterName, config.Name)
```

Comment:
it is wrong, NodePool names can't be different from upstream. Using %+v as formatting verb is really bad idea.